packages/iris-grid/src/IrisGridRenderer.ts

```diff
@@ -257,8 +257,7 @@ class IrisGridRenderer extends GridRenderer {
 
     const x = allColumnXs.get(hoverSelectColumn);
     const columnWidth = allColumnWidths.get(hoverSelectColumn);
-    assertNotNull(x);
-    assertNotNull(columnWidth);
+    if (x == null || columnWidth == null) return;
```

Comment:
I'm concerned how we got into this state. The reason there were asserts there is it should _never_ be the case that we have a `hoverSelectColumn` but no matching columnX/columnWidth in the metrics. How are we getting into this state?

Reply:
`onWheel` in `IrisGridColumnSelectMouseHandler` doesn't update `hoverSelectColumn` so if you were hovering column 0 for example, as you scroll, `hoverSelectColumn` stays `0` but eventually `allColumnXs` and `allColumnWidths` will not have `0` as a key anymore as column 0 isn't in view. Should `hoverSelectColumn` be updated in `onWheel` as well? This would result in it highlighting while the user is scrolling rather than needing to move cursor afterwards.

Reply:
Yes - we're updating the cursor, we should update the hover column as well.